examples/terraform/modules/profiles/cl/bootkube-controller.yaml.tmpl

```diff
@@ -1,6 +1,7 @@
 ---
 systemd:
   units:
+    {{ if eq .etcd_on_host "true" }}
```

Comment:
Equality comparison was required because terraform always passes strings (rather than booleans). Go template if conditions evaluate string's that are non-zero length to be truthy.